content/en/docs/v3.5/install.md

```diff
@@ -115,3 +145,5 @@ For a slightly more involved sanity check of your installation, see
 [tagged-release]: https://github.com/etcd-io/etcd/releases/tag/{{< param git_version_tag >}}
 [Supported platforms]: {{< relref "op-guide/supported-platform" >}}
 [Bitnami's etcd Helm chart]: https://bitnami.com/stack/etcd/helm
+[Arch User Repository (AUR)]: <https://wiki.archlinux.org/title/Arch_User_Repository>
```

Comment:
Just need to remove refs to AUR then I think we are good to merge 🙂

Reply:
Right, I forgot to remove it :sweat_smile:.